### PR TITLE
DDCNL-6518 - TAMC-FRONTEND - PTA single GG account

### PR DESCRIPTION
**Pull request by IanPGodman**
# Pull Request Checklist
- Add/update Welsh translation if required.
- Add any logging needed for Bridge Team monitoring.
- Add unit/acceptance/performance tests for new features.
- Make sure code coverage has not dropped.
- Run the [tamc-acceptance-tests](https://github.com/hmrc/tamc-acceptance-tests).
- Run the [tamc-performance-tests](https://github.com/hmrc/tamc-performance-tests).
- Perform relevant accessibility checks.
- Test manually in browser if required.
- Make any environment configuration changes/checks if needed.

### CODE REVIEW

File: app/connectors/MarriageAllowanceConnector.scala [hunk 1]
```diff
@@ -69,7 +69,9 @@ class MarriageAllowanceConnector @Inject()(httpClient: HttpClient, applicationCo
         case response if response.status == 200 => Right(response.json.as[GetRelationshipResponse])
         case errorResponse => Left(errorResponse.json.as[MarriageAllowanceError])
       }.recoverWith {
-      case t: Throwable => Future.failed(t)
+          case t: Throwable => {
```
Comment (Tateni93) [inline app/connectors/MarriageAllowanceConnector.scala:72]: In Scala when we have a function expression like this case match; if the change is on line of code we don't need to add braces round the code. See line 70

Comment (IanPGodman) [reply app/connectors/MarriageAllowanceConnector.scala:72]: Removed braces

File: test/views/multiyear/transfer/TransferTest.scala [hunk 2]
```diff
@@ -438,7 +442,7 @@ class TransferTest extends BaseTest with NinoGenerator {
       val todayPlusOneDay = LocalDate.now().plusDays(1)
       val pattern = "d MM YYYY"
       val todayPlusOneDayFormatted = todayPlusOneDay.format(DateTimeFormatter.ofPattern(pattern))
-      val localDate = LocalDate.now().plusYears(1)
+      val localDate = LocalDate.now().plusYears(2)
```
Comment (Tateni93) [inline test/views/multiyear/transfer/TransferTest.scala:445]: This feels like we're testing a change of behaviour here? Is this test not working at 1 year

Comment (IanPGodman) [reply test/views/multiyear/transfer/TransferTest.scala]: Test code not removed in error, corrected.

**Review by Tateni93 — APPROVED** [review]

**Pull request merged**